source/bricklaunch

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
+#
+# bricklaunch.sh <ops> <program>
+# - open a new vt and switch to it
+# - give 'user' ownership of the vt
+# - run the given programm on the new vt as 'user'
+#
+
+OPTIND=1
+user='user'
```

Comment:
Ha, I never saw that and honestly for the ev3 it might make sense to just have a default to make things simple.